spec/requests/api/v1/applications_controller_spec.rb

```diff
@@ -1,27 +0,0 @@
-require "rails_helper"
```

Comment:
having a think about those, I think I'd prefer to keep them but let's discuss in person

Reply:
bring them back with authentication added